src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -25,6 +25,9 @@ Copyright (c) .NET Foundation. All rights reserved.
           Should be re-visited when multiple TFM support is added to Dependencies logic.
     -->
     <DefaultImplicitPackages>Microsoft.NETCore.App;NETStandard.Library</DefaultImplicitPackages>
+
+    <!-- Set RuntimeIdentifier graph for NuGet -->
+    <RuntimeIdentifierGraphPath Condition="'$(RuntimeIdentifierGraphPath)' == ''">$(BundledRuntimeIdentifierGraphFile)</RuntimeIdentifierGraphPath>
```

Comment:
With this in targets, is there a good way to disable this in the csproj? Do we need one? Suppose I want to use a newer one from a package...

Reply:
I've moved it to .props now